### ISSUE TITLE
New repo for tracking projects/ecosystem?

### ISSUE THREAD

**Issue by HadrienGardeur** [readium/architecture#89]
Right now, all active projects are listed in https://readium.org/architecture/projects.html but it's hard to follow that page without getting additional "noise" tied to commits on this repo.

We might want to consider creating a separate repo just for that. IIIF has a very nice resource that might be an inspiration for that: https://github.com/IIIF/awesome-iiif

**Comment by jccr** [comment]
Would like to see an actual awesome-readium in the future too.

**Comment by rkwright** [comment]
Note that there is a repo for "planning" though its naming may not be great.

[https://github.com/readium/readium-planning](https://github.com/readium/readium-planning)

**Comment by jccr** [comment]
The [awesome](https://github.com/sindresorhus/awesome/blob/master/awesome.md) project is a curated list of curated lists.

We can create our own "awesome-readium" curated list of exemplars in the Readium ecosystem

Submitting our awesome list to _the_ awesome list is optional, we can consider that once this gets formalized.

**Comment by jccr** [comment]
Examples:
- D3: https://github.com/wbkd/awesome-d3#readme
- React: https://github.com/enaqx/awesome-react#readme
- Drupal: https://github.com/nirgn975/awesome-drupal#readme
